_includes/CSGLabsContributers.html

```diff
@@ -0,0 +1,6 @@
+<div class="container CSGLabs-temp">
```

Comment:
There's a much easier, Jekyll way to do this. Here's an example: https://github.com/jekyll/example/blob/gh-pages/about.md